### PR TITLE
Adjust fee contract

### LINKED ISSUE #12: Adjust holderg fee distribution contract
- Adjust fee contract to account for new split

### PR DESCRIPTION
**Pull request by ldgaetano**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,5 +12,30 @@ ErgoScript contracts for the Phoenix implementation of HodlCoin.
...
 
+## Fee Implementation Guidelines
+For the hodlCoin protocol, any implementation of the fee contract must send a minimum amount to Dr. Bruno Woltzenlogel Paleo's address: 9gnBtmSRBMaNTkLQUABoAqmU2wzn27hgqVvezAC9SU1VqFKZCp8
+
+- Let Q be the amount that must be sent to Bruno's address.
+- Let F be the bank fee percentage.
+- Let D be the developer fee percentage.
+- Then, Q >= 0.025 * (F/D)
+
+As long as this condition is satisfied, you may include any other constrain in your implementation of the fee contract.
```
Comment (Zahnentferner) [inline README.md]: constrain => constraint

File: contracts/phoenix_fee_contract/v1/ergoscript/phoenix_v1_hodlcoin_fee.es [hunk 2]
```diff
@@ -25,60 +25,58 @@
...
+    val pulsarzNum: Long = 25L
+    val phoenixNum: Long = 25L
+    val kushtiNum: Long  = 15L
+    val krasNum: Long    = 10L
+    
+    val brunoAddress: SigmaProp   = PK("9gnBtmSRBMaNTkLQUABoAqmU2wzn27hgqVvezAC9SU1VqFKZCp8")
+    val pulsarzAddress: SigmaProp = PK("9hHondX3uZMY2wQsXuCGjbgZUqunQyZCNNuwGu6rL7AJC8dhRGa")
+    val phoenixAddress: SigmaProp = PK("9iPs1ujGj2eKXVg82aGyAtUtQZQWxFaki48KFixoaNmUAoTY6wV")
+    val kushtiAddress: SigmaProp  = PK("9iE2MadGSrn1ivHmRZJWRxzHffuAk6bPmEv6uJmPHuadBY8td5u")
+    val krasAddress: SigmaProp    = PK("")
```
Comment (Zahnentferner) [inline contracts/phoenix_fee_contract/v1/ergoscript/phoenix_v1_hodlcoin_fee.es]: Address missing here.

Comment (ldgaetano) [reply contracts/phoenix_fee_contract/v1/ergoscript/phoenix_v1_hodlcoin_fee.es]: Yeah we are waiting for him to get back to us with an address he wants to use. Will update with new commit once we get it.

File: README.md (hunk 1 above)
Comment (ldgaetano) [reply README.md]: Thanks!

**Pull request merged**